src/controllers/authenticationController.ts

```diff
@@ -78,6 +85,15 @@ export default class AuthenticationController {
     }
 
     const spidUser = errorOrUser.value;
+
+    // If the user isn't an adult a forbidden response will be provided
+    if (
+      !fromNullable(spidUser.dateOfBirth)
+        .map(isAdult)
```

Comment:
you can use `exists(isAdult)` see https://github.com/gcanti/fp-ts/blob/1.12.0/src/Option.ts#L303

Reply:
Fixed in 34282a4